packages/sdk-js/test/json.test.ts

```diff
@@ -258,4 +261,14 @@ describe("json test suite", () => {
       );
     });
   });
+
+  it.each((cases as Cases).urlRedirect)(
+    "urlRedirect[%#] %s",
+    (name, ctx, exp, result) => {
+      const growthbook = new GrowthBook(ctx);
+      const res = growthbook.run(exp);
```

Comment:
Don't use run since we're testing auto experiments. Grab results via deferred tracking callbacks instead